sanguk/db/migrations/20221107153358_posts.sql

```diff
@@ -0,0 +1,13 @@
+-- migrate:up
+    CREATE TABLE posts (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    postingId INT NOT NULL,
+    postingImageUrl VARCHAR(500) NOT NULL,
+    postingContent VARCHAR(1000) NOT NULL,
```

Comment:
```suggestion
    image_url VARCHAR(500) NOT NULL,
    content VARCHAR(1000) NOT NULL,
```
- 테이블 내 컬럼명을 작성할 때에는 `언더바(_)`를 활용하여 `snake case`로 작성해 주세요.
- 또한 모든 네이밍 컨벤션이 동일해야 합니다. (CamelCase, snake_case를 섞어서 사용하면 안됨)

Reply:
해당 부분에 대해서 다시 한번 생각해보겠습니다. 주신 내용에 대해서는 수정 하였습니다.